packages/core/status/core-status-server-internal/src/routes/status.ts

```diff
@@ -88,7 +88,7 @@ export const registerStatusRoute = ({
         // ROUTE_TAG_ACCEPT_JWT from '@kbn/security-plugin/server' that cannot be imported here directly.
         tags: ['api', 'security:acceptJWT'],
         access: 'public', // needs to be public to allow access from "system" users like k8s readiness probes.
-        description: `Get Kibana's current status.`,
+        summary: `Get Kibana's current status.`,
```

Comment:
We're recommending not to use an ending period in the summary
```suggestion
        summary: `Get Kibana's current status`,
```